From 8ecbc1b774ffcbd38ce0bdabb2382bff2cc5c31c Mon Sep 17 00:00:00 2001
From: Alex Van Damme <atv@google.com>
Date: Mon, 24 Jun 2019 14:02:52 -0700
Subject: [PATCH 4/6] Quiet down a noisy print

- Redirect this print into the Gstreamer debug logs instead of stdout.

Change-Id: I3162b26657bf58bce585ad55ef96a61fc36f2128
---
 ext/wayland/gstwaylandsink.c | 2 +-
 sys/kms/gstkmssink.c         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ext/wayland/gstwaylandsink.c b/ext/wayland/gstwaylandsink.c
index 442d3d033..265c466aa 100644
--- a/ext/wayland/gstwaylandsink.c
+++ b/ext/wayland/gstwaylandsink.c
@@ -476,7 +476,7 @@ gst_wayland_sink_change_state (GstElement * element, GstStateChange transition)
       g_clear_object (&sink->pool);
       
       if (sink->run_time > 0) {
-        g_print ("Total showed frames (%lld), playing for (%"GST_TIME_FORMAT"), fps (%.3f).\n",
+        GST_DEBUG ("Total showed frames (%lld), playing for (%"GST_TIME_FORMAT"), fps (%.3f).\n",
                 sink->frame_showed, GST_TIME_ARGS (sink->run_time),
                 (gfloat)GST_SECOND * sink->frame_showed / sink->run_time);
       }
diff --git a/sys/kms/gstkmssink.c b/sys/kms/gstkmssink.c
index 95c477939..6855a8e6f 100644
--- a/sys/kms/gstkmssink.c
+++ b/sys/kms/gstkmssink.c
@@ -1640,7 +1640,7 @@ gst_kms_sink_change_state (GstElement * element, GstStateChange transition)
     }
     case GST_STATE_CHANGE_READY_TO_NULL:
       if (self->run_time > 0) {
-        g_print ("Total showed frames (%lld), playing for (%"GST_TIME_FORMAT"), fps (%.3f).\n",
+        GST_DEBUG ("Total showed frames (%lld), playing for (%"GST_TIME_FORMAT"), fps (%.3f).\n",
                 self->frame_showed, GST_TIME_ARGS (self->run_time),
                 (gfloat)GST_SECOND * self->frame_showed / self->run_time);
       }
-- 
2.26.0

